src/rv32_constopt.c

```diff
@@ -757,7 +756,7 @@ CONSTOPT(csw, {})
 CONSTOPT(caddi, {
     if (constopt_info->is_constant[ir->rd]) {
         constopt_info->is_constant[ir->rd] = true;
```

Comment:
Not sure why should we assign `true` to a variable that we already know that it was true in the if statement. For example:

 ```cpp
 if (constopt_info->is_constant[ir->rd]) {
        constopt_info->is_constant[ir->rd] = true;
```

It just looks like `constopt_info->is_constant[ir->rd] = true` is a redundant operation. Or do I miss something?